src/test/resources/terraform/postgres.tf

```diff
@@ -0,0 +1,58 @@
+# Public IP to be granted access to the DB
+variable "public_ip" {
+  type        = string
+  description = "Public IP Address to be granted access to database"
+  default     = "0.0.0.0"
+}
+
+# Versions of Postgres to create. 
+variable "postgresVersion" {
+  type        = list(string)
+  description = "Postgres Database Engine Version (example: 9.6, 10, 11)"
+  default     = ["9.6", "10", "11", "12", "13"]
```

Comment:
Is the idea to set up all databases in the first phase and then parallel the test phase? I think this is where they time spent waiting would be. Does this run in serial or parallel?

Reply:
That's  correct.  Terraform by default has a parallelism of n=10.  I couldn't create the databases on separate runners due to state locking.  It would require multiple workspaces.